### ISSUE TITLE
BUG: 

### ISSUE THREAD

**Issue by ksdavidc** [zsviczian/obsidian-excalidraw-plugin#980]
Bug reports just seems to be my favorite place these days. 

I hope this is just me.

**Your environment**
SYSTEM INFO:
	Obsidian version: v1.1.9
	Installer version: v0.14.15
	Operating system: Darwin Kernel Version 21.6.0: Sun Nov  6 23:31:16 PST 2022; root:xnu-8020.240.14~1/RELEASE_X86_64 21.6.0
	Login status: not logged in
	Insider build toggle: off
	Live preview: on
	Legacy editor: off
	Base theme: dark
	Community theme: Friendly
	Snippets enabled: 5
	Restricted mode: off
	Plugins installed: 239
	Plugins enabled: 195
		1: Kanban v1.5.1
		2: Admonition v9.0.5
		3: Paste URL into selection v1.7.0
		4: Telegraph Publish v1.2.3
		5: Imgur Plugin v2.2.1
		6: File Tree Alternative Plugin v2.2.8
		7: Annotator v0.2.9
		8: obsidian markmind v1.6.4
		9: Mind Map v1.1.0
		10: Contextual Typography v2.2.4
		11: PDF Highlights v0.0.4
		12: Obsidian42 - WordNet Dictionary v2.0.1
		13: Dialogue v1.0.2
		14: Emoji Toolbar v0.3.2
		15: Link Favicons v1.8.1
		16: Map of Content v0.1.18
		17: Show Current File Path v0.5.2
		18: Copy Block Link v1.0.4
		19: Journey v0.0.7
		20: Status Bar Pomodoro Timer v0.1.13
		21: Copy as Latex v0.1.9
		22: Obsidian matrix v1.0.0
		23: CardBoard v0.6.7
		24: Obsidian Big Calendar v0.3.0
		25: RSS Reader v1.2.2
		26: Gallery v0.5.8
		27: File Explorer Note Count v1.2.0
		28: Maximise Active Pane v0.0.3
		29: Better footnote v1.0.1
		30: Extract url content v0.12.1
		31: Extended MathJax v0.4.0
		32: TagFolder v0.15.11
		33: Pocket v0.8.1
		34: Footlinks v0.1.7
		35: Core Search Assistant v0.9.1
		36: Dangling links v0.3.0
		37: Indentation Lines v1.0.1
		38: Timeline v1.1.0
		39: Multi-Column Markdown v0.7.6
		40: Find and replace in selection v1.0.1
		41: Code Editor Shortcuts v1.11.0
		42: Open vault in VSCode v1.2.1
		43: Add links to current note v0.0.9
		44: txt as md v0.0.1
		45: Theme Picker v1.0.8
		46: Tag Page Preview v0.0.12
		47: Tidy Footnotes v0.1.1
		48: Key Promoter v1.2.2
		49: Taskbone OCR v1.0.0
		50: Advanced new file v1.5.0
		51: Natural Language Dates v0.6.1
		52: Obsidian42 - Text Transporter v1.0.3
		53: Better Command Palette v0.15.0
		54: Obsidian42 - Jump-to-Date v1.0.14
		55: Code Block Enhancer v1.0.5
		56: Underline v1.2.1
		57: Advanced Cursors v0.5.1
		58: Mark and Select v1.0.2
		59: Cursor Location v0.1.5
		60: Pomodoro Plugin v0.0.2
		61: Pluck v1.0.1
		62: Regex Find/Replace v1.2.0
		63: ABC Music Notation v1.2.0
		64: macOS Keyboard Navigation v1.1.2
		65: Wikipedia v1.0.3
		66: Org Mode v1.3.3
		67: Topic Linking v0.1.7
		68: Text to Speech v0.5.1
		69: Podcast Note v1.3.2
		70: Carry-Forward v1.4.4
		71: Flexible Pomodoro For Obsidian v0.6.0
		72: Copy as HTML v1.0.6
		73: Language Translator v1.0.3
		74: Second Window v1.0.5
		75: Related Notes Finder v1.1.0
		76: Open Link With v0.1.8
		77: Markdown Attributes v1.2.2
		78: Text Snippets v0.1.2
		79: Style Settings v0.4.12
		80: MySnippets v1.2.3
		81: css snippets v0.1.2
		82: Snippet Commands v0.0.2
		83: Meld Calc v1.0.2
		84: Obsidian Trello v1.6.1
		85: Tag & Word Cloud v1.3.2
		86: Dataview v0.5.47
		87: Heatmap Calendar v0.5.2
		88: Zotero Integration v2.2.34
		89: Extract PDF Annotations v1.0.4
		90: ExcaliBrain v0.1.11
		91: Open with v0.3.1
		92: Paste Mode v5.0.0
		93: Chesser v0.2.1
		94: File Info Panel v1.2.5
		95: Nomnoml Diagram v1.1.0
		96: Itinerary v1.3.0
		97: HTML Tags Autocomplete v0.0.6
		98: Lumberjack 🪓 🪵 v1.3.0
		99: Kroki v1.2.3
		100: Find & Replace in Selection v1.1.0
		101: obsidian-title-index v0.4.0
		102: Persistent Graph v0.1.5
		103: Link Headers Directly v1.0.2
		104: CookLang Editor v0.3.0
		105: Reveal Active File Button v2.0.2
		106: Hover External Link Plugin v1.0.0
		107: Fountain v0.2.1
		108: Stopwatch Plugin v0.0.8
		109: Link Archive v0.2.0
		110: Scales and Chords v1.0.1
		111: Adjacency Matrix Maker v0.6.6
		112: Limelight v1.1.0
		113: Power Search v0.0.13
		114: Reset Font Size v1.0.2
		115: MetaEdit v1.7.2
		116: koncham workspace v0.0.3
		117: Local File Interface v0.1.0
		118: Plaintext v0.2.0
		119: Furigana v0.0.1
		120: Sentence Navigator v1.2.0
		121: Pipe tricks v1.1.0
		122: Stenography v1.0.3
		123: Phone to Roam to Obsidian v1.0.1
		124: Omnisearch v1.9.1
		125: Full Calendar v0.9.5
		126: Settings Search v1.2.0
		127: Creases v0.6.6
		128: Digital Garden v2.31.0
		129: Templater v1.16.0
		130: Calendar v1.5.10
		131: Note Folder Autorename v0.1.2
		132: Folder Note Core v1.3.5
		133: Quote of the Day v1.0.1
		134: AidenLx's Folder Note v0.16.4
		135: Folder Note v0.7.3
		136: AidenLx's Folder Note - folderv component v1.0.0
		137: Emoji Shortcodes v2.2.0
		138: Icon Shortcodes v0.9.7
		139: Obsidian42 - BRAT v0.6.36
		140: Bartender v0.5.7
		141: Customizable Sidebar v2.2.1
		142: Hide Sidebars on Window Resize v1.0.4
		143: Sidebar Expand on Hover v1.0.1
		144: Liquid Templates v0.3.1
		145: Juggl v1.4.0
		146: Hover Editor v0.11.8
		147: Local REST API v1.5.1
		148: Better File Link v1.1.3
		149: Buttons v0.4.19
		150: Note Auto Creator v1.3.1
		151: Rich Links v1.1.0
		152: Text expand v0.11.2
		153: Remotely Save v0.3.25
		154: Completr v3.1.0
		155: Longform v2.0.1
		156: File Explorer Markdown Titles v1.0.2
		157: Simple Embeds v1.14.0
		158: JavaScript Init v1.0.2
		159: Markdown Formatting Assistant v0.4.1
		160: Table of Contents v0.1.3
		161: Obsidian Enhancing Export v1.1.7
		162: Pandoc Plugin v0.4.1
		163: Scratchblocks v0.0.1
		164: Link Embed v2.0.3
		165: Embed Code File v1.2.0
		166: Calibre v1.0.6
		167: Vantage - Advanced search builder v1.4.2
		168: Note Refactor v1.7.1
		169: Fleeting Notes Sync v0.4.2
		170: Obsidian Columns v1.4.0
		171: Hard Breaks v0.1.1
		172: Breadcrumbs v3.6.4
		173: Quick Explorer v0.2.7
		174: Dynamic Table of Contents v0.0.27
		175: Frontmatter Tag Suggest v0.4.1
		176: ObsidianTweaks v0.4.0
		177: Outliner v4.2.1
		178: Google Photos v1.1.9
		179: Tag Wrangler v0.5.6
		180: MAKE.md v0.5.6
		181: Commander v0.4.9
		182: cMenu v1.1.30
		183: Hotkeys++ v0.2.7
		184: Hotkey Helper v0.3.15
		185: Keyboard Analyzer v0.1.3
		186: External Link Helper v0.8.0
		187: QuickAdd v0.8.0
		188: Markdown Furigana v1.2.3
		189: Advanced Slides v1.19.0
		190: Editor Syntax Highlight v0.1.3
		191: Dynamic Highlights v0.3.1
		192: Hypothes.is v0.1.19
		193: Dynamic Embed v1.1.0
		194: Excalidraw v1.8.10
		195: Translate v1.4.4

RECOMMENDATIONS:
	Custom theme and snippets: for cosmetic issues, please first try updating your theme and disabling your snippets. If still not fixed, please try to make the issue happen in the Sandbox Vault or disable community theme and snippets.
	Community plugins: for bugs, please first try updating all your plugins to latest. If still not fixed, please try to make the issue happen in the Sandbox Vault or disable community plugins.

**Describe the bug**
after running slideshow, the original line stays transparent, making it hard to find again. 

Sometimes it briefly returns and cuts out again.
Sometimes when it starts it goes blank but returns, being visible during slideshow.
I thought it had to do with some of the line settings, but that didn't seem to matter.

I have tried reloading, restarting, brand new diagram.

**To Reproduce**
see video

**Expected behavior**
The line becomes transparent on start up, and then it returns to visibility when leaving slideshow.

**Screenshots**

https://user-images.githubusercontent.com/62230004/212997691-eb43d841-3bcb-47cd-968a-56d5ca6a22bc.mov

In the vid below I turned off the theme. Several errors here, including starting with line visible and it turning off then on, so it is visible during slideshow, and starting with a transparent line, ending up with line invisible. I hope what I did is clear. 

In short the behavior seems unpredictable (to me).

https://user-images.githubusercontent.com/62230004/212999211-d03877b6-b273-4c45-a08b-8de107f103c3.mov

**Additional context**

**Comment by zsviczian** [comment]
I see you have many-many plugins. Can you reproduce this error in a vault with only Excalidraw?
Plugins are built to different levels of quality. I recommend keeping active only those plugins you truly use every day. This minimizes the risk of conflict between plugins.

**Comment by ksdavidc** [comment]
You are quite right, I should have thought of that. I'll triage first. Apologies. 

I have several vaults that are more pristine, but the particular one I was using is a kind of workhorse that I use for work (and therefore have call for a lot of functionality). So far everything seems to be working, as far as I need them, but naturally, you are right, I should have tried that first.
